packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart

```diff
@@ -15,4 +15,8 @@ class IdeTheme {
   Color backgroundColor;
   Color foregroundColor;
   double fontSize;
+
+  double fontSizeFactor() {
```

Comment:
nit: make this a getter as it is cheap to compute. While you are at it, make the other fields of this class final as they should have always been final.